src/routes/local.ts

```diff
@@ -1,39 +1,97 @@
-import express from 'express'
-import n2yo from './../api/n2yo/index.js'
-import SatelliteUtilities from '../utils/satelliteUtils'
-import _ from 'lodash'
-import SatelliteInterface from '../utils/types/satelliteInterface.js'
+import express from "express";
+import n2yo from "./../api/n2yo/index.js";
+import SatelliteUtilities from "../utils/satelliteUtils";
+import _ from "lodash";
+import SatelliteInterface from "../utils/types/satelliteInterface.js";
+import { object, number, string, assert } from "superstruct";
 
-const router = express.Router()
-const PROPS_TO_SEND_IN_RESPOSNE = ['_id', 'officialName', 'perigee', 'apogee', 'operator', 'operatorCountry', 'purpose']
+const router = express.Router();
+const PROPS_TO_SEND_IN_RESPOSNE = [
+  "_id",
+  "officialName",
+  "perigee",
+  "apogee",
+  "operator",
+  "operatorCountry",
+  "purpose",
+];
 
-router.get('/two-line-elements/:id', (req, res) => {
-    if (req.params.id === undefined) return res.sendStatus(400)
-    n2yo.getTwoLineElements(req.params.id)
-        .then(result => res.json(result))
-        .catch(() => res.sendStatus(400))
-})
+router.get("/two-line-elements/:id", (req, res) => {
+  if (req.params.id === undefined) return res.sendStatus(400);
```

Comment:
Are you sure that a request to "/two-line-elements/" will make req.params.id undefined? What if it results in an empty string instead?